esmvalcore/preprocessor/_units.py

```diff
@@ -120,7 +121,10 @@ def convert_units(cube, units):
     return cube
 
 
-def accumulate_coordinate(cube, coordinate):
+def accumulate_coordinate(
+    cube: iris.cube.Cube,
+    coordinate: str,
```

Comment:
`coordinate` could also be in `iris.coords.Coord`. Please also remove the type hints from the docstring below, they will get added automatically

Reply:
Done in [b3d0f76](https://github.com/ESMValGroup/ESMValCore/pull/2325/commits/b3d0f7637922b54464b54bbfefd34830fb4661ea)